server/metrics/metrics.go

```diff
@@ -175,12 +175,7 @@ func New(config *libconfig.MetricsConfig) *Metrics {
 				Help:      "Total number of defunct processes in the node",
 			},
 			func() float64 {
-				total, err := process.DefunctProcesses()
-				if err == nil {
-					return float64(total)
-				}
-				logrus.Warn(err)
-				return 0
+				return float64(process.TotalDefunctProcesses())
```

Comment:
This makes the "return the current state of the system" call time-wise dependent on the zombie monitor (runs every 5 seconds for now). I'm not sure if we wanna have this coupling, how about separating both?

Reply:
yeah I was unsure but didnt' want to duplicate work. I will split it up, which will allow us to cleanup zombies less frequently